### DISCUSSION TITLE
Latest Version Mismatch in Moshi README Download Section

### DISCUSSION THREAD

**Discussion by sia-txt** [square/moshi#1789] [Q&A]
Hello,

I hope this message finds you well.

I noticed a discrepancy in the Moshi README file regarding the latest release version. The current README Download section indicates version 1.14.0, while the latest release of Moshi is 1.15.0.

I would like to inquire whether this is a simple typo or if there is a specific reason for officially recommending version 1.14.0.

Thank you for your attention to this matter.

**Comment by JakeWharton** [comment]
Typo.

Want to send a PR to fix it?

**Reply by sia-txt** [reply]
Yes. I will submit a pull request with the corrections by Wednesday.
Have a great day!
